openfga_sdk/sync/credentials.py

```diff
@@ -0,0 +1,244 @@
+"""
```

Comment:
I think we can remove this file too.

Reply:
I will remove parts of it. The `Credentials` class has some async logic that needs to be handled https://github.com/openfga/python-sdk/blob/main/openfga_sdk/credentials.py#L206C15-L206C28

Reply:
Removed what I can in https://github.com/openfga/python-sdk/pull/40/commits/2f313f464e7007d0cd4b4489a33051bba1de1882